easybuild/easyblocks/g/gromacs.py

```diff
@@ -97,7 +102,10 @@ def configure_step(self):
                 for libname in ['BLAS', 'LAPACK']:
                     lib_dir = os.getenv('%s_LIB_DIR' % libname)
                     libs = os.getenv('LIB%s' % libname)
-                    self.cfg.update('configopts', '-DGMX_%s_USER="-L%s %s"' % (libname, lib_dir, libs))
+                    if self.toolchain.toolchain_family() == toolchain.CRAYPE:
+                        self.cfg.update('configopts', '-DGMX_%s_USER="%s/libsci_gnu_mpi_mp.a"' % (libname, lib_dir))
```

Comment:
hardcoding the `_gnu_` here means that this only works with `CrayGNU`

I've fixed this in #960 with a `glob`, cfr. ec6b342673ac92d75b734cacb1e116c1f35861d8